app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self, name: str, health: int = 100, hidden: bool = False
+    ) -> None:
```

Comment:
This line breaks doesn't improve the readability of the code in my opinion. Will look better in one line, or every attribute on new line.